shippy-lib/shippy.js

```diff
@@ -259,6 +260,11 @@ let Shippy = (function() {
 		Shippy.Storage.init(); // Get files required to run this app and add them to the session storage.
 	};
 
+	window.onbeforeunload = function (e) {
```

Comment:
Make sure this is really triggered on our target browser versions. I remember I wanted to use it a couple of times and it was really unreliable.